### PR TITLE
Add multiple token test

### PR DESCRIPTION
**Pull request by pablothedude**
Add a test to validate the revocation of tokens.

### CODE REVIEW

File: tests/behat/features/bootstrap/RaContext.php [hunk 1]
```diff
@@ -290,6 +291,62 @@ public function iShouldSeeInAuditLogIdentityOverview($institution)
...
+        foreach ($searchResults as $searchResult) {
+            $button = $searchResult->find('css', '.btn.revoke');
+            if ($button) {
+                $attribute = $button->getAttribute("data-sfidentifier");
+                if ($attribute === $identifier) {
+                    $token = $attribute;
+                    break;
+                }
+            }
+        }
```
Comment (epinxteren) [inline tests/behat/features/bootstrap/RaContext.php]: Why not use find here?         $searchResult = $page->find('css', sprintf('[data-sfidentifier=%s]', json_encode($identifier)));

Comment (epinxteren) [inline tests/behat/features/bootstrap/RaContext.php]: Why not use find here?       $searchResult = $page->find('css', sprintf('.btn.revoke[data-sfidentifier=%s]', json_encode($identifier)));

File: tests/behat/features/bootstrap/RaContext.php [hunk 2]
```diff
@@ -290,6 +291,62 @@ public function iShouldSeeInAuditLogIdentityOverview($institution)
...
+        $sfIdentifier = $button->getAttribute('data-sfid');
+        $identityIdentifier = $button->getAttribute('data-sfidentityid');
+
+        // Fill data in hidden revocation form
+        $page->find('css', 'input[name="ra_revoke_second_factor[identityId]"]')->setValue($identityIdentifier);
+        $page->find('css', 'input[name="ra_revoke_second_factor[secondFactorId]"]')->setValue($sfIdentifier);
+        $form = $page->find('css', "form[name=ra_revoke_second_factor]");
+        $form->submit();
+
+        // DEBUG
```
Comment (epinxteren) [inline tests/behat/features/bootstrap/RaContext.php]: debug? Assert?

File: tests/behat/features/bootstrap/RaContext.php (hunk 1 above)
Comment (pablothedude) [reply tests/behat/features/bootstrap/RaContext.php]: see my last comment.

File: tests/behat/features/bootstrap/RaContext.php (hunk 2 above)
Comment (pablothedude) [reply tests/behat/features/bootstrap/RaContext.php]: removed

**Pull request merged**